container_images/gce-img-resource/check/check_command.go

```diff
@@ -39,10 +39,19 @@ func Run(request Request) (Response, error) {
 	}
 
 	call := computeService.Images.List(request.Source.Project)
+
+	var filter string
+
+	if request.Source.ReadyOnly {
+		filter = "(status = READY) "
+	}
+
 	if request.Source.Family != "" {
-		call = call.Filter(fmt.Sprintf("family = %s", request.Source.Family))
+		filter = filter + fmt.Sprintf("(family = %s)", request.Source.Family)
```

Comment:
Did you check if this format is correct/valid? `(status = READY)(family = family-name)` if I were to guess I'd have guessed `(status = READY, family = family-name)`.

Reply:
Nitpick: if the space between the tokens are relevant you could use a `strings.Join([]string{...}, " ")` instead.

Reply:
yep, that's the right format. Reference: https://cloud.google.com/compute/docs/reference/rest/v1/images/list

Reply:
> Nitpick: if the space between the tokens is relevant you could use a `strings.Join([]string{...}, " ")` instead.

Done